### PR TITLE
Use styled components to replace Sass

### PR DESCRIPTION
**Pull request by kevindantas**
## Summary

- Use `styled-components` to replace Sass styles.
- Match the React version to the same being used on the Portal

 _Note: This PR only replaces the components without too much logic embedded, components that have more logic embedded are still being worked alongside with the tests (e.g. Range selection, range preview)_

### CODE REVIEW

File: src/components/styles/Calendar.styled.js [hunk 1]
```diff
@@ -0,0 +1,176 @@
+import styled, { css } from 'styled-components';
+
+const marginHorizontal = css`
+  margin: 0 5px;
+`;
+
+export const CalendarWrapper = styled('div')`
```
Comment (pedrodurek) [inline src/components/styles/Calendar.styled.js]: NABD: Here, you could use the new syntax `styled.div``; `

Comment (kevindantas) [reply src/components/styles/Calendar.styled.js]: Done

**Review by pedrodurek — APPROVED** [review]

**Comment by mikelfcosta** [comment]
Just a small thing, but should we apply the same Prettier configs from our project here? The bracketSpacing is _true_ on this project.

**Comment by kevindantas** [comment]
Maybe we could add this prettier config on another PR, the PR #2 is branched out from this one, so it'll may have some conflicts if we do this right now.

**Review by mikelfcosta — APPROVED** [review]

**Pull request merged**